src/app/view/components/_common/dot-avatar/dot-avatar-component.spec.ts

```diff
@@ -72,6 +65,13 @@ describe('DotAvatarComponent', () => {
         it('should have the right src', () => {
             expect(img.nativeElement.src.endsWith(component.url)).toBe(true);
         });
+
+        it('should set src to null if path is broken', (done) => {
+            setTimeout(() => {
```

Comment:
timeout added to wait until the `GET` is done in the headless browser and in that way the `Error` handler function gets called!